examples/example-pokemon-vue/src/graphql-env.d.ts

```diff
@@ -15,17 +15,17 @@ export type introspection = {
   subscription: never;
   types: {
     'Attack': { kind: 'OBJECT'; name: 'Attack'; fields: { 'damage': { name: 'damage'; type: { kind: 'SCALAR'; name: 'Int'; ofType: null; } }; 'name': { name: 'name'; type: { kind: 'SCALAR'; name: 'String'; ofType: null; } }; 'type': { name: 'type'; type: { kind: 'ENUM'; name: 'PokemonType'; ofType: null; } }; }; };
+    'Int': unknown;
```

Comment:
nit: Would be good to check why this changed unless it's just copied over or to skip these changes